src/Spain/Mews.Fiscalizations.Spain/Model/Request/LocalCounterParty.cs

```diff
@@ -2,9 +2,9 @@
 
 namespace Mews.Fiscalizations.Spain.Model.Request
 {
-    public sealed class LocalCompany
+    public sealed class LocalCounterParty
```

Comment:
You're now saying this is not only a company. But is the TaxPayerIdentificationNumber required also for people? Is that `NifVat`?

Reply:
Yea thats the thing, this is completely incorrect, we report bills/invoices for customers and companies but here in the lib, the name "company" is being used but thats not correct.